docs/influxdb.rst

```diff
@@ -0,0 +1,70 @@
+How to use InfluxDB connector
+=============================
+
+InfluxDB is the database we use to store data and build dashboards. Our InfluxDB
+connector is in `multivac/influxdb.py`
+
+Run and connect to InfluxDB locally
+-----------------------------------
+
+1. Install InfluxDB 2 according to
+`official instruction <https://docs.influxdata.com/influxdb/v2.4/install/>`__.
+
+2. Launch InfluxDB:
+
+.. code-block:: console
+
+    $ influxd
+
+After that you will be able to work with your InfluxDB with web-interface at
+http://localhost:8086 . You can change it with
+`instruction <https://docs.influxdata.com/influxdb/v2.4/reference/urls/>`__. Go to
+your site and create the account. Save your login organisation as environmental
+variable or in `.env` as `INFLUX_ORG`
+
+3. Go to `<localhost:8086> -> Load data -> Buckets` to create bucket. Save
+bucket name to environmental variable or in `.env` file as
+`INFLUX_BUCKET=<bucket>`
+
+4. Go to `<localhost:8086> -> Load data -> API Tokens` to get your token. Save
+token to environmental variable or in `.env` file as `INFLUX_TOKEN=<token>`
+
+5. Save your InfluxDB site to environmental variable or in `.env` file as
+`INFLUX_URL`
```

Comment:
In list items, indent all lines to 4 characters.
```suggestion
3.  Go to `<localhost:8086> -> Load data -> Buckets` to create bucket. Save
    bucket name to environmental variable or in `.env` file as
    `INFLUX_BUCKET=<bucket>`

4.  Go to `<localhost:8086> -> Load data -> API Tokens` to get your token. Save
    token to environmental variable or in `.env` file as `INFLUX_TOKEN=<token>`

5.  Save your InfluxDB site to environmental variable or in `.env` file as
    `INFLUX_URL`
```

Reply:
fixed